clusterloader2/testing/load/config.yaml

```diff
@@ -3,23 +3,23 @@
 # - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
 
 #Constants
+{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
+{{$TEST_THROUGHPUT := DefaultParam .TEST_THROUGHPUT 10}}
 {{$BIG_GROUP_SIZE := 250}}
 {{$MEDIUM_GROUP_SIZE := 30}}
 {{$SMALL_GROUP_SIZE := 5}}
-{{$NODES_PER_NAMESPACE := 100}}
-{{$PODS_PER_NODE := 30}}
-{{$TEST_THROUGHPUT := 10}}
-# bigRCs - 1/4 of all pods should be in big RCs.
-{{$BIG_RCS_PER_NAMESPACE := 3}}
-# mediumRCs - 1/4 of all pods should be in medium RCs.
-{{$MEDIUM_RCS_PER_NAMESPACE := 25}}
-# smallRCs - 1/2 of all pods should be in small RCs.
-{{$SMALL_RCS_PER_NAMESPACE := 300}}
 
 #Variables
 {{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
 {{$totalPods := MultiplyInt $namespaces $NODES_PER_NAMESPACE $PODS_PER_NODE}}
 {{$saturationTime := DivideInt $totalPods $TEST_THROUGHPUT}}
+# bigRCs - 1/4 of namespace pods should be in big RCs.
+{{$bigRcsPerNamespace := DivideInt (MultiplyInt $NODES_PER_NAMESPACE $PODS_PER_NODE) (MultiplyInt 4 $BIG_GROUP_SIZE)}}
```

Comment:
I don't think that's correct. This is 

"pods per namespace / 1000" - it doesn't depend on cluster size. And clearly it should.

Reply:
>  it doesn't depend on cluster size. And clearly it should.

I don't agree. At least as long as we keep the assumption that each namespace is identical.
Currently we need to define the content of namespace. Cluster size has only impact on namespace count.

We can drop the assumption that each namespace is identical, however it will make the test config much more complex.

Reply:
aah ok - beause it's just for single namespace.